roles/ocp4_workload_connectivity_link/tasks/workload.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Install Connectivity Link Operator
+  ansible.builtin.include_role:
+    name: install_operator
+  vars:
+    install_operator_action: install
+    install_operator_name: rhcl-operator
+    install_operator_namespace: openshift-operators
+    install_operator_manage_namespaces: []
```

Comment:
I actually don't know what this does without going and looking, but you are missing this in the remove_workload. Can you just verify that it is or is not needed here or there?

Reply:
100% right.  I changed this to remove.